src/util/Helpers.cpp

```diff
@@ -437,7 +439,17 @@ bool readProviderEmotesCache(const QString &id, const QString &provider,
 
         qCDebug(chatterinoCache)
             << "Loaded emote cache: " << id << "." << provider;
-        callback(doc);
+
```

Comment:
is this change important? I'd rather review this PR without this change if possible